src/index.css

```diff
@@ -0,0 +1,69 @@
+/** @format */
+
+body {
+  margin: 0;
+  background-color: rgb(241, 67, 96) !important;
+  font-family: Arial, Helvetica, sans-serif;
+  color: white !important;
+  font-weight: bold !important;
+  padding: 0 10%;
+}
+
+nav {
+  font-size: 2.5rem;
+  margin-bottom: 5%;
+  display: flex;
+  flex-direction: column;
+}
+
+input {
+  width: 50%;
+  border-radius: 5px;
+}
+
+a {
+  text-decoration: none !important;
+  color: white !important;
+  font-family: 'Courier New', Courier, monospace;
+}
+
+.overall-container {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+}
+
+.text-center {
+  text-align: center;
+}
+
+#item-container {
+  display: flex;
+  gap: 5px;
+  box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1), 0 2px 4px rgba(0, 0, 0, 0.06);
+  padding: 10px;
+}
+
+.item-details {
+  display: flex;
+  flex-direction: row;
+  gap: 5%;
+}
+
+.item-details-div {
+  margin-top: 10%;
```

Comment:
- [ ] Similarly, let's ensure that the details page matches the [original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). The layout/positioning of elements and color pattern closely matches the [template design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)).

**Your design:**        |  **The origin design:**
:-------------------------:|:-------------------------:
<img width="800" alt="Your desgin" src="https://github.com/jkanyi-web/veejayz-gen-store/assets/12745474/572541fd-101b-42ba-a041-07817f43bbcd"> |  <img width="800" alt="Home Page" src="https://github.com/Dorcas126/capstone-project-metrics-webapp/assets/12745474/531dd146-f5a9-4727-95f3-efa7e745d879">